src/views/patient/registration/RegistrationPage.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+	<Card>
+		<template #content>
+			<div class="w-30rem flex flex-column justify-content-between align-items-center" style="gap: 20px">
+				<Toast />
+				<div class="w-30rem flex flex-row flex-justify-center">
+					<div class="card w-30rem pb-4">
+						<Steps :model="items" />
+					</div>
+				</div>
+
+				<router-view v-slot="{ Component }">
+					<component
+							:is="Component"
+							:selectedOptions="selectedOptions"
+							@select-option="({ option, value }) => (selectedOptions[option] = value)"/>
+				</router-view>
+			</div>
+		</template>
+	</Card>
+</template>
+
+<script setup>
+import { ref } from "vue";
+//import { useRouter } from "vue-router";
+//import { useToast } from "primevue/usetoast";
+import Steps from "primevue/steps";
+import Toast from "primevue/toast";
+import Card from "primevue/card";
+
+//const router = useRouter();
+//const toast = useToast();
+const selectedOptions = ref({
+	vaccine: {},
+	slot: {},
+	disease: "",
+});
+const path = "/patient/registration";
+const items = ref([
+	{
+		label: "Time slot",
+		to: `${path}/slots`,
+	},
+	{
+		label: "Choose disease",
+		to: `${path}/diseases`,
+	},
+	{
+		label: "Vaccine type",
+		to: `${path}/vaccines`,
```

Comment:
Wybieranie szczepionek nie działa, a przez to nie można zarejestrować się na szczepienie.

Z tego co sprawdziłem wysyłany z frontu request jest poprawny, bo w odpowiedzi przychodzą odpowiednie szczepionki.
![obraz](https://user-images.githubusercontent.com/57873253/163729759-00163584-d99a-47d7-8003-03956182d633.png)

Reply:
Nie miałam najnowszej wersji backendu i zmieniła się od tamtej pory odpowiedź na zapytanie o dostępne szczepionki, teraz już powinno działać, mi się udało zarejestrować na szczepienie.

Reply:
Super, teraz już działa. 👍